x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -86,18 +77,10 @@ public class EsqlCapabilities {
 
     private static Set<String> capabilities() {
         List<String> caps = new ArrayList<>();
-        caps.add(FN_CBRT);
-        caps.add(FN_IP_PREFIX);
-        caps.add(FN_SUBSTRING_EMPTY_NULL);
-        caps.add(AGG_TOP_LIST);
-        caps.add(ST_CENTROID_AGG_OPTIMIZED);
-        caps.add(METADATA_IGNORED_FIELD);
-        caps.add(FN_MV_APPEND);
-        caps.add(REPEAT);
-        caps.add(NAMED_POSITIONAL_PARAMETER);
```

Comment:
I think this is missing in the enum

Reply:
And any specific reason why `STRING_LITERAL_AUTO_CASTING_TO_DATETIME_ADD_SUB` is not in the enum either? I added it, but I see it's at the end of the list, so maybe I'm missing some context

Reply:
No context, just missed it. Good catch!